gitpod-io/openvscode-server
# [gitpod-desktop] Connect using ssh gateway

Connects to gitpod workspace direclty using ssh gateway, local app is kept for backward compatibility 

Related https://github.com/gitpod-io/gitpod/issues/8513

How to test:
- uninstall remote ssh extension
- Optional, go to preferences and select vscode desktop
- Open any workspace in gitpod
- If in VSCode Browser run command Open in VSCode/VSCode Insiders
- Workspace should open in desktop local without launching local app, you'll be prompted to install remote-ssh extension
